bindings/bindings.py

```diff
@@ -0,0 +1,593 @@
+import abc
```

Comment:
can this file be called `generate_bindings.py` or something along these lines to indicate this is the generator?